deepchem/models/tensorgraph/models/graph_conv.py

```diff
@@ -0,0 +1,163 @@
+import tensorflow as tf
+from deepchem.models.tensorgraph.tensor_graph import TensorGraph
+from deepchem.models.tensorgraph.layers import Input, Dense, Concat, SoftMax, SoftMaxCrossEntropy, Layer, \
+  GraphConvLayer, BatchNormLayer, GraphPoolLayer, GraphGather, WeightedError
+from deepchem.metrics import to_one_hot
+from deepchem.feat.mol_graphs import ConvMol
+import time
+
+
+class GraphConvTensorGraph(TensorGraph):
+  """
+  """
+
+  def __init__(self, **kwargs):
+    super().__init__(**kwargs)
+    self.min_degree = 0
+    self.max_degree = 10
+
+  def _construct_feed_dict(self, X_b, y_b, w_b, ids_b):
```

Comment:
This looks totally fine for this PR, but I think we should make a generic solution to constructing feed-dicts for graphs with many inputs (DTNNs, Weaves, DAGs, etc all have many different input types).  Ideally, all the MoleculeNet models should require no custom subclasses with methods beyond those in `TensorGraph`

I have an idea for a general solution. Will post on #498.